defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+accumulo_trace_user: "root"
```

Comment:
@melrief I propose to set by default:
> accumulo_trace_password: "{{accumulo_secret}}"

also plz move the 2 new settings below the existing ones (at least behind accumulo_secret)